src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -91,6 +91,39 @@ private CSharpCompilation CreateNullableCompilation(CSharpTestSource source, IEn
             return CreateCompilation(source, options: WithNonNullTypesTrue(), references: references);
         }
 
+        [Fact, WorkItem(37456, "https://github.com/dotnet/roslyn/issues/37456")]
```

Comment:
is this the right file for this test?

Reply:
I think it's ok because the regression came from nullability work.

Reply:
Moved the test ;-)